src/alpha/AlphaStonesTab.js

```diff
@@ -65,8 +65,8 @@ export default function AlphaStonesTab({state, popup, updateState}) {
                 )}</div>)}
                 <br/>
                 {xBonus > 0 && <div style={{fontSize:"20px",fontWeight:"bold"}}>{spaces()}s<sub>x</sub> = {getStoneCalculationForX(state, stoneTable)} = {formatNumber(xBonus, state.settings.numberFormat)}<br/></div>}
-                {xBonus > 1 && <div>{spaces()} s<sub>x</sub> multiplies your Starting x<br/></div>}
-                {xBonus > 1 && <div>{spaces()} Your total Starting x is {formatNumber(getStartingX(state),state.settings.numberFormat, 2)}</div>}
+                {xBonus > 1 && <div>{spaces()} s<sub>x</sub> multiplies your starting x<br/></div>}
+                {xBonus > 1 && <div>{spaces()} Your total starting x is {formatNumber(getStartingX(state),state.settings.numberFormat, 2)}</div>}
```

Comment:
Similar to "Formula Efficiency" I consider "Starting x" to be stat/concept for the game, hence the capitalization.

Reply:
I completely understand this reasoning. However, "Starting x" also reads as an English mistake and not as a game term to me, probably because x is not capitalized. Given that you want to have X capitalized as a way to refer to all x (x, x', x'', x'''), we also cannot solve that problem by writing "Starting X". If you still want me to undo all changes and write it as "Starting x" for this pull request I will, just let me know!